apps/web/lib/editor/index.tsx

```diff
@@ -238,7 +238,6 @@ const ImageUpload = ({
   const uploadToClient = (event: { target: { files: any[] } }) => {
     if (event.target.files && event.target.files[0]) {
       const i = event.target.files[0];
```

Comment:
how about using optional chaining for them e.g.
```suggestion
    if (event.target?.files?.[0]) {
```

Reply:
done